photomap/frontend/static/javascript/curation.js

```diff
@@ -0,0 +1,391 @@
+            currentSelectionIndices = new Set();
+            data.selected_indices.forEach(idx => {
+                if (!excludedIndices.has(idx)) {
+                    currentSelectionIndices.add(idx);
+                }
+            });
+            currentSelectionFiles = data.selected_files;
+            analysisResults = data.analysis_results;
+
+            // Merge new results into Global Metadata Map
+            analysisResults.forEach(r => {
+                globalMetadataMap.set(r.index, {
+                    filename: r.filename,
+                    subfolder: r.subfolder,
+                    filepath: r.filepath, // Important: Store full path for export
+                    frequency: r.frequency,
+                    count: r.count
+                });
+            });
+
+            // Bucketize for Colors (Heatmap)
+            // We only look at items that are in the Top N Winners (currentSelectionIndices)
+            const freqMap = {};
+            data.analysis_results.forEach(r => freqMap[r.index] = r.frequency);
+
+            currentSelectionIndices.forEach(idx => {
+                if (!excludedIndices.has(idx)) {
+                    const freq = freqMap[idx] || 100;
+                    if (freq >= 90) highFreqIndices.add(idx);
+                    else if (freq >= 70) medFreqIndices.add(idx);
+                    else lowFreqIndices.add(idx);
+                }
+            });
+
+            updateVisuals();
+            exportBtn.disabled = false;
+            if (csvBtn) csvBtn.disabled = false;
+
+            const selectedCount = data.count || currentSelectionIndices.size;
+            const target = data.target_count || targetCount;
+
+            let msg = `${selectedCount} out of ${target} images selected.`;
+            if (excludedIndices.size > 0) {
+                msg += ` (${excludedIndices.size} excluded)`;
+            }
+            setStatus(msg, "success");
+
+        } catch (e) {
+            console.error(e);
+            setStatus("Error: " + e.message, "error");
+        }
+    };
+
+    // Export
+
+    exportBtn.onclick = async () => {
+        const path = document.getElementById('curationExportPath').value;
+        if (!path) { alert("Please enter path."); return; }
+
+        // Reconstruct file list from current indices, respecting exclusions
+        let filesToExport = [];
+        currentSelectionIndices.forEach(idx => {
+            if (!excludedIndices.has(idx)) {
+                const meta = globalMetadataMap.get(idx);
+                if (meta && meta.filepath) {
+                    filesToExport.push(meta.filepath);
+                }
+            }
+        });
+
+        if (filesToExport.length === 0) {
+            alert("No files selected to export (all excluded?).");
+            return;
+        }
+
+        setStatus(`Exporting ${filesToExport.length} files...`, "loading");
+        try {
+            const response = await fetch('/api/curation/export', {
```

Comment:
```suggestion
            const response = await fetch('api/curation/export', {
```
Another place where a relative API URL is needed.